Gems/Terrain/Code/CMakeLists.txt

```diff
@@ -49,8 +47,8 @@ ly_add_target(
 )
 
 # the above module is for use in all client/server types
-ly_create_alias(NAME Terrain.Servers  NAMESPACE Gem TARGETS Gem::Terrain)
-ly_create_alias(NAME Terrain.Clients  NAMESPACE Gem TARGETS Gem::Terrain)
+ly_create_alias(NAME Terrain.Servers  NAMESPACE Gem TARGETS Gem::Terrain Gem::SurfaceData.Servers Gem::GradientSignal.Servers)
```

Comment:
Does it also not need the same declaration below, for Tools / Builders?

Reply:
That is a good question, I didn't add them because the dependencies for tools were getting built and loaded (SurfaceData.editor.dll, GradientSignal.editor.dll and libs) but looking at the Terrain.Editor target I'm wondering how CMake determined the Tools variant should result in the shared SurfaceData library being built, but the Client variant didn't.

Reply:
This is most likely because another Gem TARGET that depends on the SurfaceData.Editor and GradientSignal.Editor targets.

Reply:
You're right, the LandscapeCanvas gem has build and runtime dependencies on SurfaceData.Editor, and it is in the enabled_gems.cmake list.  I'll update the other variants.